include/grpc++/impl/call.h

```diff
@@ -90,10 +93,12 @@ class CallOpBuffer : public CompletionQueueTag {
   grpc_metadata_array recv_initial_metadata_arr_;
   // Send message
   const grpc::protobuf::Message *send_message_;
-  grpc_byte_buffer *send_message_buf_;
+  const ByteBuffer *send_message_buffer_;
+  grpc_byte_buffer *send_buf_;
```

Comment:
Why you need both ByteBuffer and grpc_byte_buffer? My understanding is ByteBuffer wraps a grpc_byte_buffer.